.github/workflows/gradle.yml

```diff
@@ -102,6 +112,13 @@ jobs:
           cd build/js/packages/doordeck-sdk
           npm publish --access public
 
+      - name: Push to CocoaPods
+        env:
+          COCOAPODS_TRUNK_TOKEN: ${{ secrets.COCOAPODS_TRUNK_TOKEN }}
+        run: |
+          cd library/build/cocoapods/publish/release
+          pod trunk push DoordeckSDK.podspec --allow-warnings --verbose
```

Comment:
should we add:
```
        pod lib lint --allow-warnings
```

for lint inspection?

Reply:
Unless I'm missing something, it doesn't seem to be necessary: https://github.com/CocoaPods/CocoaPods/issues/8098#issuecomment-422177177